<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressMessageDialog</class>
 <widget class="QWidget" name="ProgressMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Please Wait...</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="m_icon">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/progress/Resources/progress/waitCursor0.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_label">
     <property name="text">
      <string>Scanning directory for iTunes database...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_cancelButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../player.qrc"/>
 </resources>
 <connections/>
</ui>
